client/coral-admin/src/components/KarmaTooltip.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { Icon } from 'coral-ui';
+import styles from './KarmaTooltip.css';
+import ClickOutside from 'coral-framework/components/ClickOutside';
+import t from 'coral-framework/services/i18n';
+
+const initialState = { menuVisible: false };
+
+class KarmaTooltip extends React.Component {
+  static propTypes = {
+    settings: PropTypes.shape({
+      reliable: PropTypes.number.isRequired,
+      unreliable: PropTypes.number.isRequired,
```

Comment:
Also here it wasn't clear for me what those props represents, maybe `thresholds: { reliable, unnreliable }` is more descriptive or even better `reliableThreshold`, `unreliableThreshold` to avoid using nested props.

Reply:
I'll change it to `thresholds` to better reflect the meaning.